src/components/crossDevice/CrossDeviceLink/style.css

```diff
@@ -101,29 +101,29 @@
 
 .linkText {
   resize: none;
-  height: 24px;
+  height: 24*@unit;
   border: none;
   background-color: #f3f3f4;
   overflow: hidden;
-  font-size: 16px;
-  line-height: 24px;
+  font-size: inherit;
```

Comment:
curious, why is the inherit necessary?

Reply:
Some input elements like `button` are annoying in that they don't default in inheriting their parents fonts

Here we're just making 100% sure that we _are_ inheriting our parent's fonts, in case we end up in such an awkward element